google-cloud-testing/storage-benchwrapper/src/main/java/com/google/cloud/benchwrapper/Main.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.benchwrapper;
+
+import java.util.Properties;
+import io.grpc.Server;
+import io.grpc.netty.NettyServerBuilder;
+
+class Main {
+  public static void main(String[] args) throws Exception {
+    Properties p = new Properties(System.getProperties());
```

Comment:
p -->properties per google style rules

Reply:
Could you point me at the relevant part of go/java-style? AFAICT no part of the variables section makes any mention of local variables (5.1, 5.2.*) needing to be words.

Reply:
Thanks. Done